examples/model_selection/plot_permutation_tests_for_classification.py

```diff
@@ -101,10 +103,10 @@
 #
```

Comment:
I can't comment high enough but above, what do you think about changing to:

```
score_label = f"Score on original\niris data: {score_iris:.2f}\n(p-value: {pvalue_iris:.3f})
```

and similarly below, "original random data" ..?

Reply:
Good idea. This clarifies it further. I have adapted this suggestion.